easybuild/easyconfigs/a/ADOL-C/ADOL-C-2.7.0-gompi-2019a.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'ConfigureMake'
+
+name = 'ADOL-C'
+version = '2.7.0'
```

Comment:
@omula since this is a commit and not a relase or tagged version, better to include the commit date in the version:
```
version = '2.7.0-20190130'
```
also, do you have a specific reason why you did not choose a more recent commit?

Reply:
@smoors in fact they labeled it as 2.7.0 although they did not create a tag.
```
Fix conflict in psdrivers.c Mladen Banovic authored 8 months ago
7022a97f
Merge branch 'svn' Mladen Banovic authored 8 months ago
75aa3541
Regenerate all files with v2.7.0. mbanovic authored 8 months ago
51c74bf3
Increase version number to 2.7 with new features. mbanovic authored 8 months ago 
77fabd7d
```

It seems that they did not create the tag/release in the svn although they did label it https://projects.coin-or.org/ADOL-C/browser

Reply:
ah, so you took the first commit with the new version number that passed the pipeline test? that's ok for me then.

Reply:
Exactly :)